exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKit.java

```diff
@@ -292,20 +295,67 @@ private void checkTransaction(TransactionMessage transactionMessage) {
    * corresponds to the latest committed block). In-pool (not yet processed) transactions are also
    * accessible with it in {@linkplain Blockchain#getTxMessages() blockchain}.
    *
+   * <p>This method destroys the snapshot once the passed closure completes, compared to
+   * {@link #getSnapshot()}, which disposes created snapshots only when closing the TestKit.
```

Comment:
Nit: I think it must say "when TestKit is closed" [by something else] because getSnapshot does not close it itself.